lib/Yield/ControlFlow/SVG.cpp

```diff
@@ -68,9 +68,9 @@ static constexpr auto ArrowHead = R"(<marker
   id="{0}"
   markerWidth="{1}"
   markerHeight="{1}"
-  refX="{1}"
+  refX="{5}"
   refY="{2}"
-  orient="{5}"><polygon
+  orient="{6}"><polygon
   points="{4}, {1} {3}, {2} {4}, {4} {1}, {2}" /></marker>
```

Comment:
Actually, after I cleaned up this stuff a bit, I was planning to move all of graph related stuff on top of `@mrjackv`'s `Tag`s.

Reply:
You sure that's a good idea? That was more tailored towards wrapping text.

Reply:
I feel like converting this into a bunch of calls to `::ptml::Tag::addMarker("refX", std::to_string(MyComputedInCppValue)` would be pretty beneficial, would it not?

Reply:
Let's try and see.

Reply:
What do you think? https://github.com/revng/revng/pull/191/commits/0dc63460e37f42b1d53ae64c972da89917ae7256

Reply:
I feel like the following makes it needlessly harder to make changes to the template compared to the previous solution.
```c++
  Border.addAttribute("class", ::tags::NodeBody)
    .addAttribute("x", std::to_string(TopLeft.X))
    .addAttribute("y", std::to_string(TopLeft.Y))
    .addAttribute("rx", std::to_string(Configuration.NodeCornerRoundingFactor))
    .addAttribute("ry", std::to_string(Configuration.NodeCornerRoundingFactor))
    .addAttribute("width", std::to_string(Node->Size.W))
    .addAttribute("height", std::to_string(Node->Size.H));
```

Reply:
I don't know. To me it's easier to read this compared to how it used to be, mostly because it's really clear what's going where and why (no need to keep track of argument indices).

If it's all the `to_string`s that you don't like, we can add a couple of additional `addAttribute` overloads to handle non-strings.

Reply:
to be extra clear, I'm talking about something like
```
template <typename T>
concept NameIsWIP = requires(T &&t) {
  { std::to_string(std::forward<T>(t)) } -> std::convertible_to<std::string_view>;
};
// . . .
class Tag {
  // . . .
  template <NameIsWIP T>
  Tag &addAttribute(std::string_view Key, T &&Value) {
    return addAttribute(Key, std::to_string(std::forward<T>(Value)));
  }
  // . . .
};
```

Reply:
On a second look, this is OK.